lib/ansible/module_utils/connection.py

```diff
@@ -36,6 +36,28 @@
 from functools import partial
 from ansible.module_utils._text import to_bytes, to_text
 from ansible.module_utils.six import iteritems
+from ansible.module_utils.six.moves import cPickle
+
+
+def write_to_socket(fd, obj):
```

Comment:
Naming wise, I don't think these are sockets so we probably don't want to call it that.  (also shuld remove socket from the comments)